code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -765,6 +765,17 @@
 		/obj/item/clothing/head/white_dress = 50,
 		/obj/item/clothing/shoes/white = 50,
 		/obj/item/clothing/gloves/white = 50,
+		/obj/item/instrument/violin = 2,
+		/obj/item/instrument/piano_synth = 2,
+		/obj/item/instrument/banjo = 2,
+		/obj/item/instrument/guitar = 2,
+		/obj/item/instrument/glockenspiel = 2,
+		/obj/item/instrument/accordion = 2,
+		/obj/item/instrument/trumpet = 2,
+		/obj/item/instrument/saxophone = 2,
+		/obj/item/instrument/trombone = 2,
+		/obj/item/instrument/recorder = 2,
+		/obj/item/instrument/harmonica = 2,
```

Comment:
Req: Marines shouldnt have to suffer req point loss for their RP
New Vendor: It would clutter the already small prep halls with a vendor only holding 10 items
On the floor: Prep is already a mess
Somewhere on the floor out of the way: PVT Unga Banana should not be forced to break into maint so he can play banjo in FOB

Reply:
If you want to fix bloat do vendor tabs and start with the hilariously full weapon vends